program/src/oracle/oracle.c

```diff
@@ -490,6 +529,11 @@ static uint64_t upd_price( SolParameters *prm, SolAccountInfo *ka )
   // update aggregate price as necessary
   if ( sptr->slot_ > pptr->agg_.pub_slot_ ) {
     upd_aggregate( pptr, sptr->slot_, sptr->unix_timestamp_ );
+    if (pptr->agg_.status_ == PC_STATUS_TRADING) {
```

Comment:
why if status==TRADING? I think it would be nice to know when the status has other values as well

Reply:
the initial thought was that the aggregate price will be useful only when its valid but we can definitely remove the check